garrysmod/gamemodes/terrortown/gamemode/init.lua

```diff
@@ -91,6 +94,8 @@ CreateConVar("ttt_det_credits_starting", "1")
 CreateConVar("ttt_det_credits_traitorkill", "0")
 CreateConVar("ttt_det_credits_traitordead", "1")
 
+-- Other
+CreateConVar("ttt_scoreboard_sorting", "1", FCVAR_NOTIFY + FCVAR_REPLICATED)

```

Comment:
Instead of a ConVar, why not allow the clients to sort by clicking on the column headings or something?